src/libaktualizr/package_manager/packagemanagerfake.cc

```diff
@@ -110,6 +110,7 @@ bool PackageManagerFake::fetchTarget(const Uptane::Target& target, Uptane::Fetch
     return false;
   }
 
+  // TODO(OTA-4939): Unify this and make it more generic.
```

Comment:
Perhaps explicitly saying unification with what exactly is desired would be helpful in future? Like function name or file for example.

Reply:
Fair point, done.